Entities/Ingame/Club.cs

```diff
@@ -48,13 +48,13 @@ public Club (int memoryAddress, ArraySegment<byte> originalBytes, IVersion versi
 
 		public string Name { 
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.Name, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.Name, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
 		public string ShortName {
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.ShortName, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.ShortName, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
```

Comment:
You are actually right. I just did a review, and it seems it has changed from 15.2.1 > 15.3.x
In this case, can you review the code so that we can set the name offset in the version file, so we can support older versions, too? The inner name offset is:

Version | Offset
------------ | -------------
`FM 14.3.1` | 0xC
`FM 15.0.x` | 0x4
`FM 15.1.x` | 0x4
`FM 15.2.x` | 0x4
`FM 15.3.x` | 0x0  

Thanks, and sorry for that, got totally confused.